src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3Connection.cs

```diff
@@ -423,7 +423,7 @@ public void RemoveStream(QuicStream stream)
                 memberName,                   // method name
                 message);                     // message
 
-        private async ValueTask SendSettingsAsync()
+        private async Task SendSettingsAsync()
```

Comment:
This one is because the call site is doing:
```C#
_ = SendSettingsAsync();
```
and thus is never waiting for the operation to complete.  If we don't go ahead with the feature to automatically pool value task instances, then returning Task here is no worse than returning ValueTask.  If we do go ahead with the feature, returning Task here is strictly better because we'd never be awaiting the ValueTask and thus never returning it to the pool.